bin/compile

```diff
@@ -1,25 +1,27 @@
 #!/usr/bin/env bash
 # bin/compile <build-dir> <cache-dir>
 
-if [ $STACK != "scalingo-18" ]
-then
-        echo "Executing legacy compile"
-        . "/buildpack/bin/compile-scalingo-14"
-        exit $?
-fi
-
 # fail fast
 set -e
 
 # debug
-# set -x
+if [ -n "$BUILDPACK_DEBUG" ]; then
+    set -x
+fi
+
+legacy_stacks="cedar-14 scalingo scalingo-14"
```

Comment:
`cedar-14` and `scalingo` can be completely removed. `scalingo` doesn't exist anymore on production regions, it's now named `scalingo-14`